mindsdb/integrations/handlers/chromadb_handler/chromadb_handler.py

```diff
@@ -471,6 +472,8 @@ def drop_table(self, table_name: str, if_exists=True):
         self.connect()
         try:
             self._client.delete_collection(table_name)
+            if "persist_directory" in self._client_config:
```

Comment:
persist_directory is always in self._client_config. is better to check if it is none

also, I think we can do only for folders that are in self.handler_storage
because you can use local folder and create two collection (tables) there
after deletion of the first table the second shouln't be deleted